ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/History/Summary.hs

```diff
@@ -331,8 +346,16 @@ summarize ::
   Transitions xs ->
   Summary xs
 summarize ledgerTip = \(Shape shape) (Transitions transitions) ->
-  Summary $ go initBound shape transitions
+  Summary $ go initBoundWithPeras shape transitions
  where
+  -- as noted in the haddock, this function is only used for testing purposes,
+  -- therefore we make the initial era is Peras-enabled, which means
+  -- we only test Peras-enabled eras. It is rather difficult
+  -- to parameterise the test suite, as it requires also parameterise many non-test functions, like
+  -- 'HF.initBound', and leads to a huge diff. Therefore, we make the judgement call to
+  -- only test Peras-enabled eras.
+  initBoundWithPeras = initBound{boundPerasRound = PerasEnabled . PerasRoundNo $ 0}
```

Comment:
I think this makes sense for now, but we might want to revisit this in the future. Can you create a corresponding issue in https://github.com/tweag/cardano-peras/issues?

Also, a similar comment also occurs in `genSummary`

Reply:
I've rephrased the comments and created the issue: https://github.com/tweag/cardano-peras/issues/112